Warp.WebApp/Controllers/ImageController.cs

```diff
@@ -46,6 +53,156 @@ public async Task<IActionResult> Upload([FromForm] List<IFormFile> images)
         return Ok(results);
     }
 
-    
+
+    [HttpPost]
+    [DisableFormValueModelBinding]
+    [ValidateAntiForgeryToken]
+    public async Task<IActionResult> UploadDatabase()
+    {
+        if (!MultipartRequestHelper.IsMultipartContentType(Request.ContentType))
+        {
+            ModelState.AddModelError("File", $"The request couldn't be processed (Error 1).");
+            // Log error
+
+            return BadRequest(ModelState);
```

Comment:
This is an API controller, return `ProblemDetails` instead of `ModelState`. See [`EntryService:29`](https://github.com/punk-link/warp/blob/9967e8a63ff1b2ed5987fa78a40b3d4082517b1c/Warp.WebApp/Services/Entries/EntryService.cs#L29)